build/Dockerfile.full

```diff
@@ -0,0 +1,50 @@
+FROM golang:1.18 as build
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download -x
+COPY . .
+RUN rm -rf /src/webconsole/dist
```

Comment:
Can this be added to .dockerignore? Or am I missing something in understanding this?

Reply:
I see the same approach is used in based dockerfile

Reply:
I don't even have a `/webconsole/dist` directory on my machine... I don't know why this is done (I don't know how webconsole is built) but what you say make totally sense.

Reply:
Fixed in commit 5ca03f68
I guess we can fix also other dockerfiles

Reply:
I agree with you, thanks for fixing this Simone, I'll try to add it in other dockerfiles too